u2f-ref-code/java/src/com/google/u2f/server/data/SecurityKeyData.java

```diff
@@ -149,7 +149,8 @@ public String toString() {
       .append(counter)
       .append("\n")
       .append("attestation certificate:\n")
```

Comment:
Nit: is this label misleading now? Should it say "attestation certificate (chain):\n"? Or do you want to try to be smarter about what to call it?